litestar/_openapi/responses.py

```diff
@@ -280,7 +280,11 @@ def create_error_responses(exceptions: list[type[HTTPException]]) -> Iterator[tu
                         "status_code": Schema(type=OpenAPIType.INTEGER),
                         "detail": Schema(type=OpenAPIType.STRING),
                         "extra": Schema(
-                            type=[OpenAPIType.NULL, OpenAPIType.OBJECT, OpenAPIType.ARRAY],
+                            any_of=[
+                                Schema(type=OpenAPIType.NULL),
+                                Schema(type=OpenAPIType.OBJECT),
+                                Schema(type=OpenAPIType.ARRAY, items=Schema()),
```

Comment:
Should the `items` be `Schema()` or a `Reference` to the created schema?

Reply:
I believe here items is just specifying what is expected to be contained within the array. Since we're trying to type it as Any this should work. This [section on arrays](https://swagger.io/docs/specification/data-models/data-types/#array) says:  

> An array of arbitrary types can be defined as: 
```
type: array
items: {}
```
> Here, {} is the “any-type” schema.

Reply:
Oh yeah, you're absolutely right. I have no idea what I was thinking when I thought it should be the reference to the created schema :P